packages/generator/src/generateFontDirectoryPackage.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
+import fsExtra from 'fs-extra';
+import path from 'path';
+
+import { PackageScope, FontDirectoryPackageDir } from './constants';
+import { FontItem } from './types';
+import { createFileFromTemplate } from './utils/createFileFromTemplate';
+import { getNextPackageVersion } from './utils/getNextPackageVersion';
+import { getPackageNameForWebfont } from './utils/name';
+
+export async function generateFontDirectoryPackage(
+  fonts: FontItem[],
+  options?: { patch?: boolean }
+) {
+  const version = getNextPackageVersion(FontDirectoryPackageDir, options);
+  await fsExtra.emptyDir(FontDirectoryPackageDir);
+
+  // Clone the object
+  const fd = JSON.parse(JSON.stringify(fonts));
+
+  for (const webfont of fd) {
+    const packageName = getPackageNameForWebfont(webfont);
+    webfont.expoGoogleFontsPackage = PackageScope + packageName;
+    webfont.expoGoogleFontsPackageHomepage = `https://github.com/expo/google-fonts/tree/main/font-packages/${packageName}`;
+  }
+
+  await fs.promises.writeFile(
+    path.join(FontDirectoryPackageDir, 'fontDirectory.json'),
+    JSON.stringify(fd),
```

Comment:
We should wrap the fd with `{"kind":"webfonts#webfontList","items": ... }` to not introduce any breaking changes or release it as meaningful change
<img width="1425" alt="image" src="https://github.com/user-attachments/assets/39df7e43-c4e8-4743-9985-205265d50e9c" />

Reply:
Good spot!